pkg/userdata/userdata.go

```diff
@@ -126,6 +127,7 @@ func (data *UserData) NewIdentityCSR() (csr *x509.CertificateSigningRequest, err
 	names := []string{hostname}
 	opts = append(opts, x509.DNSNames(names))
 	opts = append(opts, x509.IPAddresses(ips))
+	opts = append(opts, x509.NotAfter(time.Now().Add(87600*time.Hour)))
```

Comment:
Now that I'm looking at this, this option is only valid for the certificate creator.